tests/components/xiaomi_miio/test_vacuum.py

```diff
@@ -463,7 +470,7 @@ async def test_xiaomi_vacuum_fanspeeds(hass, caplog, mock_mirobo_fanspeeds):
         {"entity_id": entity_id, "fan_speed": "invent"},
         blocking=True,
     )
-    assert "ERROR" in caplog.text
+    assert "Fan speed step not recognized" in caplog.text
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)